test/views/declaration/summary/SupportingDocumentsViewSpec.scala

```diff
@@ -76,9 +76,9 @@ class SupportingDocumentsViewSpec extends UnitViewSpec with ExportsTestData with
 
     "display all supporting documents without change buttons" when {
 
-      "mode is Print" in {
+      "actionsEnabled is false" in {
```

Comment:
This case is already tested staring from line 106. So you can remove this block.